cli/azd/pkg/exec/command_runner.go

```diff
@@ -252,6 +252,55 @@ func appendEnv(env []string) []string {
 	return nil
 }
 
+// logBuilder builds messages for running of commands.
+type logBuilder struct {
```

Comment:
I'd recommend moving `logBuilder` out to its own file to make it easier to maintain over time.

Reply:
Trying to not over-expose the functionality here. It's meant to be an internal helper for `command_runner.Run(:?List)?`